content/en/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -41,6 +41,13 @@ controller can't delete it because the finalizer exists. When the Pod stops
 using the `PersistentVolume`, Kubernetes clears the `pv-protection` finalizer,
 and the controller deletes the volume.
 
+{{<note>}}
+* When you `DELETE` an object, Kubernetes makes that object almost read-only with exception that the 
```

Comment:
Is it true that you can't patch an object that is pending deletion? For example, does initating deletion prevent updates to `.status`?

Reply:
What happens if you try to edit the object using its `status` subresource? I actually don't know if objects pending deletion have their status locked to immutable.

Reply:
> Kubernetes makes that object almost read-only

This is not accurate. Kubernetes does prevent adding new finalizers but does not generally tighten modification restrictions. Fields under spec/status can still generally be changed as they could prior to deletionTimestamp being set.